fedimint-api/src/bitcoin_rpc.rs

```diff
@@ -1 +1,31 @@
+use std::ffi::OsStr;
 
+use url::Url;
+
+/// Name of the env value used for passing bitcoind rpc url to modules that need it
+pub const FM_BITCOIND_RPC_ENV: &str = "FM_BITCOIND_RPC";
+
+/// Default url that will be used if [`FM_BITCOIND_RPC_ENV`] is not set
+pub const FM_BITCOIND_RPC_DEFAULT_FALLBACK: &str = "http://127.0.0.1:8332";
```

Comment:
why default to mainnet?

Reply:
i guess this is the default of bitcoin-cli ...

just seem like kinda arbitrary default.

Reply:
Because that's the intended setup it's meant to be used with. Anyone not running it against mainnet, is a developer, not user, and they do an extra step, I think.

If anyone has good arguments against it, we can change. :)

Reply:
Can't argue with that!